lib/app/features/chat/presentation/talk/chat_main_talks_page.dart

```diff
@@ -12,14 +11,24 @@ import '../pages/chat_channel_card.dart';
 import 'chat_main_talks_controller.dart';
 
 class ChatMainTalksPage extends StatefulWidget {
-  const ChatMainTalksPage({Key? key}) : super(key: key);
+  const ChatMainTalksPage({Key? key, required this.controller})
+      : super(key: key);
+
+  final ChatMainTalksController controller;
 
   @override
   _ChatMainTalksPageState createState() => _ChatMainTalksPageState();
 }
 
-class _ChatMainTalksPageState
-    extends ModularState<ChatMainTalksPage, IChatMainTalksController> {
+class _ChatMainTalksPageState extends State<ChatMainTalksPage> {
+  ChatMainTalksController get controller => widget.controller;
+
+  @override
+  void initState() {
+    super.initState();
+    controller.initialize();
```

Comment:
Melhor garantir que o controller seja acionado apenas quando as Widgets estiverem prontas para isto

```suggestion
    WidgetsBinding.instance.addPostFrameCallback((_) {
      controller.initialize();
    });
```